### PR TITLE
 fix delete user resource bug

### LINKED ISSUE #59: Bug when deleting users with same prefix
I have created 2 users: test and test2
if I delete test2 user everything is ok and test user is still available
if I delete test user, test2 user is gone from cluster but still it's showing in permission manager and if I click on username in permission manager application hangs. I've created rolebinding manually in order to bring back user and make application work again.

Same behavior was for 2+ users with the following names:
user
user-dev
user-test
user-staging
user-prod
I deleted user and all other users(rb,sa) were deleted from cluster but they still exist in permission-manager and when I click on one of them the app hangs.

### PR DESCRIPTION
**Pull request by Rampage1xx**
Hello, 

This PR aims to close #59

It's a join work of me and @phisco

### CODE REVIEW

File: internal/resources/user.go [hunk 1]
```diff
@@ -101,7 +101,9 @@ func (r *resourcesService) CreateUser(ctx context.Context, username string) User
 // the PermissionManagerUser CRD object associated to the user with the given username.
 func (r *resourcesService) DeleteUser(ctx context.Context, username string) {
 	metadataName := "permissionmanager.user." + username
-	_, err := r.kubeclient.AppsV1().RESTClient().Delete().AbsPath(resourceURL + "/" + metadataName).DoRaw(ctx)
+
+	_, err := r.kubeclient.AppsV1().RESTClient().Delete().AbsPath(resourceURL + "/" + metadataName + "/").DoRaw(ctx)
+
```
Comment (phisco) [inline internal/resources/user.go:106]: Was this needed in the end?

Comment (Rampage1xx) [reply internal/resources/user.go]: tested without, it still works, so im gonna remove it

File: web-client/src/constants.tsx [hunk 2]
```diff
@@ -63,6 +63,9 @@ export const VERBS = [
   'delete',
 ]
 
-export const templateNamespacedResourceRolePrefix = 'template-namespaced-resources___'
+export const resourceSeparator = '___'
```
Comment (phisco) [inline web-client/src/constants.tsx:66]: can you use it also [here](https://github.com/sighupio/permission-manager/blob/master/web-client/src/services/rolebindingCreateRequests.tsx#L147) and [here](https://github.com/sighupio/permission-manager/blob/master/web-client/src/services/rolebindingCreateRequests.tsx#L168)?

**Review by phisco — APPROVED** [review]
LGTM 🚀

**Pull request merged**